pkg/scaffold/v1/resource/resource_test.go

```diff
@@ -119,6 +119,12 @@ var _ = Describe("Resource", func() {
 			Expect(instance.Resource).To(Equal("cats"))
 		})
 
+		It("should allow webapp-project as a Group", func() {
```

Comment:
test names should describe the intent of the test, so this should say something like "should allow hyphens in group names".

Reply:
Done 👍 Tks for your input.